.scalafix.conf

```diff
@@ -3,7 +3,8 @@ rules = [
     DisableSyntax,
     ProcedureSyntax,
     NoValInForComprehension,
-    LeakingImplicitClassVal
+    LeakingImplicitClassVal,
+    ExplicitResultTypes
```

Comment:
`ExplicitResultTypes` rule is the ci-error reason.
IMO, it is a useful rule. But we need an upgrade version from `scalafix-maven-plugin`.
I have a new pull request for it: https://github.com/evis/scalafix-maven-plugin/pull/15 (waiting approval from the maintainer)

Reply:
Can we open 2 PRs?

 -> One without ExplicitResultTypes that we can merge pretty much straight away
 -> And a second one that eventually adds ExplicitResultTypes rule but depends on your external contribution to be merged and released?

Reply:
Ah that's the ExplicitResultTypes then that need to match the scala version? Ok let's go without it first then